stubs/app/client/ui/HomeImports.ts

```diff
@@ -1,13 +1,13 @@
+import {AppModel, reportError} from 'app/client/models/AppModel';
```

Comment:
Moving code to `stubs` is undesirable. The `stubs` directory is intended for material that is just enough to convert what is in core to a working complete build, but that will be replaced in other flavors of Grist. So any code that is common to all builds should live in core. Only small wrappers should live in `stubs`.

That said, development here has been chaotic and I'm not sure if it has been coherent. Developers working on our SaaS are not particularly aware of other builds or what `stubs` is for since it doesn't affect them, so there could be a nest of weirdness here of people making things work somehow or other.